app/presenters/machine_readable/transaction_faq_page_schema.rb

```diff
@@ -0,0 +1,156 @@
+class MachineReadable::TransactionFaqPageSchema
+  attr_reader :transaction
+
+  # Only relevant for /register-to-vote at present
+  CONTENT_IN_SCOPE = %w(
+    834a7921-260b-4061-9de1-edda3e998c68
+  ).freeze
+
+  def self.build(transaction)
+    if CONTENT_IN_SCOPE.include?(transaction.content_id)
+      new(transaction)
+    end
+  end
+
+  def initialize(transaction)
+    @transaction = transaction
+  end
```

Comment:
You could move methods + constants like `initialize` and `CONTENT_IN_SCOPE` under `private`.

Reply:
Good shout, have done so.